wireguard/docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: wireguard_app_1
+      APP_PORT: 51821
+  
+  app:
+    image: ghcr.io/wg-easy/wg-easy:12@sha256:daa94b77901bdef3cb10151416c095e64cf66a6539fa8f748c09812b60b97f49
+    restart: on-failure
+    stop_grace_period: 1m
+    ports:
+      - "51820:51820/udp"
+    volumes:
+      - ${APP_DATA_DIR}/data/wireguard:/etc/wireguard
```

Comment:
Can you please add this host path with a `.gitkeep` as described here (https://github.com/getumbrel/umbrel-apps/pull/1083#discussion_r1609144592)?